app/templates/index.hbs

```diff
@@ -1,7 +1,17 @@
-<section class="lead">
-  <h2 class="text-center">EmberCamp is coming to Chicago!</h2>
-  <p>Join us on September 21 for the first-ever EmberCamp Chicago, where we’ll be hosting 150+ of the world’s top Ember
-    developers for a full day of Ember talks, all on the 56th floor of the historic Chase Tower in downtown Chicago.
+{{!-- <section class="lead">
+  <h2 class="text-center">EmberCamp Chicago 2019!</h2>
+</section> --}}
+
+<section>
+  <h2 class="ribbon ribbon--left ribbon--flipped ribbon--centered">Announcement Signup</h2>
+  <p class="text-center">Want to know when tickets go on sale? Add your email below!</p>
+  <p class="cta"><a class="button" target="_blank" href="https://goo.gl/forms/PAxPYOAYm2pVAxrF3">Sign Up for Announcements</a></p>
+</section>
+
+<section>
+  <h2 class="ribbon ribbon--right ribbon--flipped ribbon--centered">Details</h2>
+  <p>Join us on August 23rd for the second annual EmberCamp Chicago, where we’ll be hosting 150+ of the world’s top Ember
```

Comment:
should we include the year 2019 somewhere to make it clear that it's been updated?

Reply:
yup! Updated